wiki/Beatmap_Spotlights/en.md

```diff
@@ -22,7 +22,7 @@ The Beatmap Spotlights project is run by various community members across all ga
 | Role | Members |
 | :-- | :-- |
 | Project lead | ![][flag_PL] [Venix](https://osu.ppy.sh/users/5999631) |
-| Project manager | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) |
+| Project manager | ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) |
```

Comment:
if you update this article again, I think it'd be good to explain what the difference between these two roles are. or pick better words to get it across